indico/htdocs/js/palette.js

```diff
@@ -27,4 +27,9 @@
 
     // colors for specific purposes
     Palette.highlight = Palette.blue;
+
+    // colors for the bubble avatar in abstract page
+    global.avatarPalette = ['#E06055', '#FF8A65', '#E91E63', '#F06292', '#3F51B5', '#BA68C8', '#7986CB', '#5E97f6',
```

Comment:
Can we use all uppercase or (even better imo) all lowercase for the hex chars and not mix both please? :)